front/src/app/styleguide/styleguide.component.html

```diff
@@ -14,3 +14,7 @@
   </mat-form-field>
 </form>
 
+<div>
+  <button>Тестовый запуск</button>
+</div>
+
```

Comment:
???

Reply:
I except you to self-review the code right after PR creation, please avoid leaving some testing stuff